modules/siddhi-core/src/main/java/org/wso2/siddhi/core/ExecutionPlanRuntime.java

```diff
@@ -159,6 +166,10 @@ public InputHandler getInputHandler(String streamId) {
         return eventSourceMap.values();
     }
 
+    public ExecutionPlanContext getExecutionPlanContext() {
```

Comment:
Do we need to expose ExecutionPlanContext? 
This will be unnecessary for a normal user. Can't we avoid this?